packages/keystone/lib/Keystone/index.js

```diff
@@ -23,11 +23,6 @@ const {
 const { SessionManager } = require('@keystonejs/session');
 const { AppVersionProvider, appVersionMiddleware } = require('@keystonejs/app-version');
 
-const {
-  unmergeRelationships,
-  createRelationships,
-  mergeRelationships,
-} = require('./relationship-utils');
```

Comment:
The `relationship-utils` module isn't used anywhere else and should also be removed in this PR.